Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/ApplePayTokenizationViewModel.swift

```diff
@@ -205,7 +205,10 @@ class ApplePayTokenizationViewModel: PaymentMethodTokenizationViewModel {
 
                 do {
                     let session = AppState.current.apiConfiguration!.clientSession!
-                    orderItems = try self.createOrderItemsFromClientSession(session)
+                    let applePayOptions = PrimerAPIConfiguration.current?.paymentMethods?
```

Comment:
Could we encapsulate this check inside `createOrderItemsFromClientSession`? Then this callsite stays the same and we can manage the logic to use either ClientSession or PrimerSettings there

Reply:
We can, seemed to me that it could be valuable to know that `createOrderItemsFromClientSession` will receive additional context from `Configuration.paymentMethods.options`.  If your suggestion is preferred approach, I am happy to change it.

Reply:
Its ok to leave it as it is - I can see the value in being explicit